### PR TITLE
Heroku hosting changes

### PR DESCRIPTION
**Pull request by apham1**
Changes made to allow Heroku to host code directly from GitHub:

- Added Procfile and app.json files.

### CODE REVIEW

File: FoodKeeper-API/build.gradle [hunk 1]
```diff
@@ -15,8 +15,9 @@ apply plugin: 'org.springframework.boot'
 apply plugin: 'io.spring.dependency-management'
 
 bootJar {
-    baseName = 'gs-rest-service'
-    version =  '0.1.0'
+    baseName = 'foodkeeper-rest-service'
+    mainClassName = 'com.github.LibreFoodPantry.FoodKeeperAPI.App'
+    version =  '0.0.1'
```
Comment (kwurst) [inline FoodKeeper-API/build.gradle]: Why was 0.0.1 chosen as the version?

Is this not ready to be released as version 1 yet?

Comment (kwurst) [reply FoodKeeper-API/build.gradle]: Ahh.... I see it is from my comment to #5.

We should discuss versioning. How complete do you feel this is?

Comment (apham1) [reply FoodKeeper-API/build.gradle]: I think it's mostly done, but maybe I should finish up the landing page before giving it a 1.0.0 version number. 

Also, I'm not certain if all the endpoints are correct. For example, the endpoint "/products" returns the entire list of products, which I'm not sure is necessary.

Comment (kwurst) [reply FoodKeeper-API/build.gradle]: Let's bump the version to 1.0.0. 

If I can change this directly on GitHub, I will, the merge it.

Comment (kwurst) [reply FoodKeeper-API/build.gradle]: Nope. I can't do it on GitHub.

@apham1 Please bump the version to 1.0.0, push the change, then I'll merge it.

Comment (apham1) [reply FoodKeeper-API/build.gradle]: Fixed in commit 2fd8776

**Comment by kwurst** [comment]
Closed #5

**Pull request merged**